test/Admin.Test/UnitTest1.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Admin.Test;
```

Comment:
💭 What can we put here that prompts inquiry for adding tests? I am thinking we rename the file, class, and initial method to something implying "placeholder" or "temporary", perhaps even with a build warning.

Reply:
I can rename the tests to `PlaceholderUnitTest` if we want.